index.js

```diff
@@ -8,16 +8,24 @@ app.use(express.urlencoded({extended: true}));
 app.disable('etag');
 // app.use(morgan('combined'));
 
-/* app.use((req,res,next) => {
-    console.log("req.params =",req.params);
-    console.log("req.headers =",req.headers);
-    console.log("req.body =",req.body);
+if (process.env.NODE_ENV == 'development') {
+  app.use((req, res, next) => {
+    console.log('req.query =', req.query);
+    // console.log('req.params =', req.params);
+    // console.log('req.headers =', req.headers);
+    console.log('req.body =', req.body);
     next();
-}); */
+  });
+}
 
-morganBody(app, {
-  logAllReqHeader: true,
-});
+let morganBodyConfig = {};
+if (process.env.NODE_ENV == 'development') {
```

Comment:
In production environment also, we should have logs so that it can help debug any issues if they occur, what do you think?

Reply:
Okay, sounds right.